File: Python/List-2/sum67.py
def sum67(nums):
  sum = 0
  found_6  = False
  
  for i in range(len(nums)):
    if (nums[i] == 6):
      found_6 = True
    
    if not found_6:
      sum += nums[i]
    else:
      if (nums[i] == 7):
        found_6 = False
  
  return sum

File: Java/Array-2/sum67.java
public int sum67(int[] nums) {
  boolean seen6 = false;
  int count = 0;
  for (int i = 0; i < nums.length; i++) {
    if (nums[i] == 6) {
      seen6 = true;
    }
    if (!seen6) {
      count += nums[i];
    }
    if (nums[i] == 7) {
      seen6 = false;
    }
  }
  return count;
}
